src/UIKit/UIView.cs

```diff
@@ -176,6 +176,15 @@ public static Task<bool> AnimateAsync (double duration, NSAction animation)
 		{
 			return AnimateNotifyAsync (duration, animation);
 		}
+
+		public UIImage Capture (bool afterUpdates = false)
+		{
+			UIGraphics.BeginImageContextWithOptions (Bounds.Size, Opaque, 0.0f);
```

Comment:
minor (but since it needs an update anyway) you could use `var bounds = Bounds;` to prevent another round trip into ObjC code :-)